src/app/store/index.ts

```diff
@@ -22,7 +23,7 @@ if (isDevEnv) {
 export const setupStore = (): ReturnType<typeof configureStore> => {
   return configureStore({
     reducer: rootReducer,
-    middleware: (getDefaultMiddleware) => (getDefaultMiddleware() as GetDefaultMiddlewareType).concat(...middleware),
+    middleware: (getDefaultMiddleware) => (getDefaultMiddleware() as GetDefaultMiddlewareType).concat(middleware, bannerApi.middleware),
```

Comment:
To fix error add all apis like this

middleware: (getDefaultMiddleware) => (getDefaultMiddleware() as GetDefaultMiddlewareType).concat(
      ...middleware,
      baseApi.middleware as MiddlewarePointType,
      authApi.middleware as MiddlewarePointType,
      bannerApi.middleware as MiddlewarePointType
    ),